lib/tasks/locations.rake

```diff
@@ -0,0 +1,49 @@
+namespace :location do
+  desc 'Create a new location'
+  task create: :environment do
+    class MissingParamError < StandardError; end
+
+    begin
+      user = User.find_by!(email: ENV['USER_EMAIL'] || raise(MissingParamError))
```

Comment:
Can't we just unpack the environment variables using `fetch` with no default? That would raise `KeyError` that you could handle. Or even better unpack all of the required environment variables early on and have a guard instead, something like:

```ruby
unless email && organisation && title && address_line_1 && postcode
  abort '... Usage'
end
```

I used `abort` specifically, as this will write to `STDERR` and exit with a failure code.